packages/molenc/molenc.0.0.1/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/molenc"
+bug-reports: "https://github.com/UnixJunkie/molenc/issues"
+dev-repo: "git+https://github.com/UnixJunkie/molenc.git"
+license: "BSD-3"
+build: ["dune" "build" "-p" name "-j" jobs]
+install: ["cp" "bin/molenc_type_atoms.py" "%{bin}%/molenc_type_atoms.py"]
+remove: ["sh" "-c" "rm -f %{bin}%/molenc_type_atoms.py"]
```

Comment:
```suggestion
remove: ["rm" "-f" "%{bin}%/molenc_type_atoms.py"]
```
I don't see any need for `sh` here

Reply:
```suggestion
```
Alternatively the `remove` field can also be entirely discarded as opam 2 does track the files installed by the package.